components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -163,7 +163,10 @@ tweaking their $case/namelist_scream.xml file.
     </homme>
 
     <!-- P3 microphysics -->
-    <p3 inherit="physics_proc_base"/>
+    <p3 inherit="physics_proc_base">
+      <do_prescribed_ccn>true</do_prescribed__ccn>
```

Comment:
I still see some double underscores here. There's one in the second separator in the closing tag of `</do_prescribed__ccn>` and both of the second separators for `do_predic__nc`

Reply:
ha - I was fixing 2 fast 2 furious, but not quite as cool